src/packages/utils/webpack.prod.config.js

```diff
@@ -1,21 +1,24 @@
 const webpack = require("webpack");
 const path = require("path");
+// uncomment to analyze
+// const BundleAnalyzerPlugin = require("webpack-bundle-analyzer")
+//   .BundleAnalyzerPlugin;
```

Comment:
Use it or remove it.. we don't want commented out code in the codebase.

Reply:
We need this plugin but only when we want to analyze the bundle so pushed it behind an env variable.